opennlp-tools/src/main/java/opennlp/tools/langdetect/Language.java

```diff
@@ -36,4 +43,30 @@ public String getLang() {
   public double getConfidence() {
     return confidence;
   }
+
+  @Override
+  public String toString() {
+
+    return getLang();
```

Comment:
We should show the confidence here as well, just imagine taking a look at this object in the debugger.